### DISCUSSION TITLE
resolved

### DISCUSSION THREAD

**Discussion by ghost** [satijalab/seurat#5476] [Q&A]

**Comment by samuel-marsh** [comment]
Hi,

What do you mean by off center?
What assay are you using for DE analysis?

Best,
Sam

**Comment by samuel-marsh** [comment]
You can see here regarding running DE following SCT workflow:
https://github.com/satijalab/seurat/issues/2180#issuecomment-546444032

Also if you are running this comparing clusters to each other you should be aware of the inherent issues in cluster vs cluster DE and that as such setting p value of significance is confounded by inflated p values.

Best,
Sam

**Reply by ghost** [reply]
Thanks Sam
